woocommerce/woocommerce-android
# IAP:  Expose acknowledgement status from `isWPcomPlanPurchased` method

Closes: #7891


### Description

While working on the documentation I realized that it might be important to know for the client if a purchase was acknowledged. This PR adds exposure of this status

### Testing instructions

Using showcase activity
* Check if the plan purchase
* Purchase a plan
* Check if the plan purchase again and notice that it shows that the plan purchase but not acknowledged
Acknowledging will be done via a backend call

https://user-images.githubusercontent.com/4923871/202696622-0b42527e-3c79-4a3a-aa0c-7165ef589aa6.mp4
